lib/async/cursorAsyncGenerator.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
nit: can we stick to the same file naming conventions used throughout the module? I know it's not the greatest, but I don't think a world where we have mixed conventions feels any better.

Reply:
also fwiw it looks like it would be like `async_iterator.js` or `cursor_async_iterator.js` rather than an async generator

Reply:
The function returned from this file is a generator, not an iterator. If you like, we could call it `cursor_asyncIterator_generator.js`

Reply:
eh, but I called it asyncIterator. I'll rename it to `get_async_iterator.js`